services/core-web/src/styles/settings/filepond-overrides.scss

```diff
@@ -10,6 +10,10 @@
     .filepond--drop-label {
         min-height: 84.6px;
     }
+
+    .filepond--credits {
```

Comment:
Noticed that in some cases where the file upload was right above the submit button, that the credits half-overlapped the submit button, so I would try to submit and end up opening the filepond documentation. Probably came with updating filepond.